packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -18,18 +17,29 @@ interface CyCookie {
   httpOnly: boolean
 }
 
+// Cypress uses the webextension-style filtering
+// https://developer.chrome.com/extensions/cookies#method-getAll
+type CyCookieFilter = chrome.cookies.GetAllDetails
```

Comment:
why not import the `chrome` types at the top vs putting them in the `compilerOptions`? I feel like this comes out of nowhere

Reply:
if i do `import 'chrome'`, build fails

if i do `import chrome from 'chrome'`, it says 'chrome' is not a module

i can dig into why `import 'chrome'` makes the build fail if you'd like, i'm not sure this is the best way either

Reply:
ask @tgriesser in #typescript channel

Reply:
nvm he's on his way to the airport - @tgriesser or @thlorenz can you explain why this is necessary whenever you have time?